rest/taskana-rest-spring/src/main/java/pro/taskana/rest/MonitorController.java

```diff
@@ -46,37 +47,57 @@
         @RequestParam(required = false) List<TaskState> states) throws NotAuthorizedException,
         InvalidArgumentException {
         LOGGER.debug("Entry to getTasksStatusReport()");
+        ReportResource report = reportAssembler.toResource(
+            taskMonitorService.createTaskStatusReportBuilder().stateIn(states).domainIn(domains).buildReport(),
+            domains, states);
         if (LOGGER.isDebugEnabled()) {
```

Comment:
This if statement is not necessary.

Reply:
This is a convention for logging, if the object could be a big object it only will be shown in debug level.